frontend/src/cards/PopulationCard.tsx

```diff
@@ -85,22 +85,28 @@ export function PopulationCard(props: PopulationCardProps) {
                 <Grid
                   container
                   className={styles.PopulationCard}
-                  justify="space-around"
+                  justify="flex-start"
+                  alignItems="flex-start"
                 >
                   <Grid item>
                     <span>Total Population</span>
                     <span className={styles.TotalPopulationValue}>
                       {totalPopulationSize}
                     </span>
                   </Grid>
-                  {/* TODO- calculate median age */}
+                  {/* TODO- calculate median age 
```

Comment:
Is this functional?  Can we uncomment it?

Reply:
This was part of the original mocks but we didn't have the data value available. We're de-prioritizing it for launch but I wanted to leave it here as a reminder that we do want to add it eventually